ParameterDataApi.yaml

```diff
@@ -2,10 +2,10 @@
 openapi: 3.0.1
 info:
   title: FRAIHMWORK Parameter Data API
-  description: 'FRAIHMWORK allows for parameter time-series data to be provided from the client. This API will be versioned according to SemVer 2.0 rules (as described here: https://semver.org/)'
+  description: 'FRAIHMWORK allows for time series or single sample parameter data to be provided from the client. This API will be versioned according to SemVer 2.0 rules (as described here: https://semver.org/)'
   contact:
     email: matt.synborski@resilienx.com
```

Comment:
General Question: Do we have a "service" email for contact purposes instead of using a specific person's email?

Reply:
This is a good idea. Out of scope for this change, but I'll put in the virtual office. I think a support@resX would be appropriate.